CONTRIBUTING.md

```diff
@@ -43,6 +43,40 @@ Documentation found on the [Microcosm site](http://code.viget.com/microcosm) is
 
 You may want to run what you see on [code.viget.com/microcosm](http://code.viget.com/microcosm) locally. To do so head over the [site section of this repo](site).
 
+## Prettier
+
+We are using [prettier](https://github.com/prettier/prettier) combined with [eslint](http://eslint.org/) to keep formatting and format linting easy. We do that by running:
+
+```bash
+yarn run pretty
+```
+
+or
+
+```bash
+make pretty
+```
+
+These will first run `prettier` to format our code and then run `eslint --fix` to make additional style changes and fixes.
+
+### Things to note about the prettier/eslint combo
+
+We are defaulting to prettier's opinions on some things (the ones it forces us to). Most notably single quotes. Prettiers **will** use double quotes (even in single quote mode) when it is "prettier" to do so.
+
+ie:
+```javascript
+// prettier will do this (what we chose to do)
+"I am using double quotes because it's prettier"
+
+// eslint --fix would do this
+'I am using single quotes because it\'s eslint --fix'
+```
+
+We are letting this happen for two reasons:
+
+1. Write whatever you want, prettier will fix it, that's the point
+2. Our `yarn run lint` task would error out if we did a `eslint --fix`. There is no way to turn change prettier's behaviour but we can turn off the offending eslint rule
+
```

Comment:
@nhunzaker this is new. While changing the npm script for `lint` I ran into some problems which led me to change some things. I summarized in this bit of the docs.